admin-dev/themes/new-theme/scss/pages/_product_page.scss

```diff
@@ -1050,3 +1050,19 @@
     }
   }
 }
+
+#form_step2_specific_price_sp_id_customer-data {
+  border: rem(1px) solid #e5e5e5;
```

Comment:
We don't use use the `rem()` mixin anymore, it's deprecated by Bourbon.
Please, directly put the rem values.